generators/server/templates/src/main/resources/config/_application-prod.yml

```diff
@@ -216,5 +216,5 @@ jhipster:
             enabled: false
             # edit spring.metrics.export.delay-millis to set report frequency
     <%_ } _%>
-    swagger: # swagger is disabled. It can be disabled by pasing 'no-swagger' profile at run time as well
-        enabled: false
+    swagger: # swagger is disabled by default in prod, it can be enabled by passing the 'swagger' profile at run time
+        enabled: #swagger.enabled#
```

Comment:
if you want this to be set, then you need to add it to gradle and pom in same way the default profiles are set. But since now you can do this purely with profile may be this doesn't have any use